schema.graphql

```diff
@@ -32,6 +33,9 @@ type Tree @entity {
   id: ID!
   childOfTree: Tree
   linkedToHat: Hat
+  requestedLinkToTree: Tree
```

Comment:
Since this Tree is the entity that made the request, this name makes sense here.